docs/production/install.md

```diff
@@ -44,23 +42,21 @@ sudo -s  # If not already root
 This takes a few minutes to run, as it installs Zulip's dependencies.
 For more on what the installer does, [see details below](#installer-details).
 
-If the script gives an error, consult [Troubleshooting](#troubleshooting) below.
+If the script returns an error, consult [Troubleshooting](#troubleshooting) below.
 
 #### Installer options
 
 - `--email=you@example.com`: The email address of the person or team
   who should get support and error emails from this Zulip server.
-  This becomes `ZULIP_ADMINISTRATOR` ([docs][doc-settings]) in the
-  Zulip settings.
+  This becomes `ZULIP_ADMINISTRATOR` ([docs][doc-settings]) in the Zulip settings.
 
 - `--hostname=zulip.example.com`: The user-accessible domain name for
   this Zulip server, i.e., what users will type in their web browser.
-  This becomes `EXTERNAL_HOST` ([docs][doc-settings]) in the Zulip
-  settings.
+  This becomes `EXTERNAL_HOST` ([docs][doc-settings]) in the Zulip settings.
 
 - `--self-signed-cert`: With this option, the Zulip installer
-  generates a self-signed SSL certificate for the server. This isn't
-  suitable for production use, but may be convenient for testing.
+  generates a self-signed SSL certificate for the server.
+- This isn't suitable for production use, but may be convenient for testing.
```

Comment:
This change is wrong; we don't want an additional bullet that would appear to be another option.